### ISSUE TITLE
<raw> tag error

### ISSUE THREAD

**Issue by Blake32p** [maizzle/framework#194]
- Maizzle Version: 1.0.9
- Node.js Version: 13.10.1

I'm having an issue with raw tags maizzle. I believe this was fixed in this version but I can't seem to get it to work. I'm not sure if I'm using them incorrectly?

Here is an example of a view as webpage link I'm using 

`<a href="<raw>{{system.viewAsWebpageLink}}</raw>" target="_blank">View Online</a>`

I get this error

`✖ Failed to compile build_local/module-template.html
(node:23887) UnhandledPromiseRejectionWarning: ReferenceError: system is not defined
(node:23887) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:23887) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`

If I remove the `<raw>` tags and link it builds correctly.

**Comment by cossssmin** [comment]
You can't use a tag inside an attribute, you need to use `@{{ }}` - read the [docs](https://maizzle.com/docs/templates/#curly-brace-conflicts).

**Comment by Blake32p** [comment]
Thank you for the quick reply, and for explaining that. I still seem to have an issue when using `@{{ }}` with that link. 

`@{{system.viewAsWebpageLink}}` wont work `@{{systemviewAsWebpageLink}}` - Without the period does work.

[https://github.com/Blake32p/raw-test](https://github.com/Blake32p/raw-test)

**Comment by cossssmin** [comment]
Looks like it won't work like that _in components_.

Try this instead:

```html
<!-- Inside an attribute -->
<raw>
  <a href="@{{ system.viewAsWebpageLink }}" target="_blank">View Online</a>
</raw>

<!-- Inside a tag -->
<raw>
  <a href="#" target="_blank">@{{ system.viewAsWebpageLink }}</a>
</raw>
```

**Comment by cossssmin** [comment]
This should be fixed in [v1.1.1](https://github.com/maizzle/framework/releases/tag/v1.1.1), please update and test - it should work as you wanted initially. Feel free to reopen if you're still encountering issues.
